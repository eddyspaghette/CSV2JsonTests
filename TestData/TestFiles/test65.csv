1,
2,102,201,301,401,501
3|104,202,302,402,502
4,106,203,303,403,503
5,108,204,304,404,504
6,110,205,305,405,505
7,112,206,306,406,506
8,114,207,307,407,507
2,102,201,301,401,501
3,104,202,302,402,502
4,106,203|303,403,503
12,122,211,311,411,511
13,124,212,312,412,512
14,126,213,313,413,513
15,128,214,314,414,514
16,130;215,315,415,515
17,132,216,316,416,516
18,134,217,317,417,517
19,136,218,318,418,518
20,138,219,319,419,519
21,140,220,320,420,520
22,142,221,321,421,521
23,144,222,322,422,522
24,146,223,323,423,523
2,102,201,301,401,501
3,104,202,302,402,502
4,106,203,303,403,503
