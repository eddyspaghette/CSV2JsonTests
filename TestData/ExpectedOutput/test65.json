[
 {
   "1": "2",
   "FIELD2": "102",
   "FIELD3": "201",
   "FIELD4": 301,
   "FIELD5": 401,
   "FIELD6": 501
 },
 {
   "1": "3|104",
   "FIELD2": "202",
   "FIELD3": "302",
   "FIELD4": 402,
   "FIELD5": 502,
   "FIELD6": null
 },
 {
   "1": "4",
   "FIELD2": "106",
   "FIELD3": "203",
   "FIELD4": 303,
   "FIELD5": 403,
   "FIELD6": 503
 },
 {
   "1": "5",
   "FIELD2": "108",
   "FIELD3": "204",
   "FIELD4": 304,
   "FIELD5": 404,
   "FIELD6": 504
 },
 {
   "1": "6",
   "FIELD2": "110",
   "FIELD3": "205",
   "FIELD4": 305,
   "FIELD5": 405,
   "FIELD6": 505
 },
 {
   "1": "7",
   "FIELD2": "112",
   "FIELD3": "206",
   "FIELD4": 306,
   "FIELD5": 406,
   "FIELD6": 506
 },
 {
   "1": "8",
   "FIELD2": "114",
   "FIELD3": "207",
   "FIELD4": 307,
   "FIELD5": 407,
   "FIELD6": 507
 },
 {
   "1": "2",
   "FIELD2": "102",
   "FIELD3": "201",
   "FIELD4": 301,
   "FIELD5": 401,
   "FIELD6": 501
 },
 {
   "1": "3",
   "FIELD2": "104",
   "FIELD3": "202",
   "FIELD4": 302,
   "FIELD5": 402,
   "FIELD6": 502
 },
 {
   "1": "4",
   "FIELD2": "106",
   "FIELD3": "203|303",
   "FIELD4": 403,
   "FIELD5": 503,
   "FIELD6": null
 },
 {
   "1": "12",
   "FIELD2": "122",
   "FIELD3": "211",
   "FIELD4": 311,
   "FIELD5": 411,
   "FIELD6": 511
 },
 {
   "1": "13",
   "FIELD2": "124",
   "FIELD3": "212",
   "FIELD4": 312,
   "FIELD5": 412,
   "FIELD6": 512
 },
 {
   "1": "14",
   "FIELD2": "126",
   "FIELD3": "213",
   "FIELD4": 313,
   "FIELD5": 413,
   "FIELD6": 513
 },
 {
   "1": "15",
   "FIELD2": "128",
   "FIELD3": "214",
   "FIELD4": 314,
   "FIELD5": 414,
   "FIELD6": 514
 },
 {
   "1": "16",
   "FIELD2": "130;215",
   "FIELD3": "315",
   "FIELD4": 415,
   "FIELD5": 515,
   "FIELD6": null
 },
 {
   "1": "17",
   "FIELD2": "132",
   "FIELD3": "216",
   "FIELD4": 316,
   "FIELD5": 416,
   "FIELD6": 516
 },
 {
   "1": "18",
   "FIELD2": "134",
   "FIELD3": "217",
   "FIELD4": 317,
   "FIELD5": 417,
   "FIELD6": 517
 },
 {
   "1": "19",
   "FIELD2": "136",
   "FIELD3": "218",
   "FIELD4": 318,
   "FIELD5": 418,
   "FIELD6": 518
 },
 {
   "1": "20",
   "FIELD2": "138",
   "FIELD3": "219",
   "FIELD4": 319,
   "FIELD5": 419,
   "FIELD6": 519
 },
 {
   "1": "21",
   "FIELD2": "140",
   "FIELD3": "220",
   "FIELD4": 320,
   "FIELD5": 420,
   "FIELD6": 520
 },
 {
   "1": "22",
   "FIELD2": "142",
   "FIELD3": "221",
   "FIELD4": 321,
   "FIELD5": 421,
   "FIELD6": 521
 },
 {
   "1": "23",
   "FIELD2": "144",
   "FIELD3": "222",
   "FIELD4": 322,
   "FIELD5": 422,
   "FIELD6": 522
 },
 {
   "1": "24",
   "FIELD2": "146",
   "FIELD3": "223",
   "FIELD4": 323,
   "FIELD5": 423,
   "FIELD6": 523
 },
 {
   "1": "2",
   "FIELD2": "102",
   "FIELD3": "201",
   "FIELD4": 301,
   "FIELD5": 401,
   "FIELD6": 501
 },
 {
   "1": "3",
   "FIELD2": "104",
   "FIELD3": "202",
   "FIELD4": 302,
   "FIELD5": 402,
   "FIELD6": 502
 },
 {
   "1": "4",
   "FIELD2": "106",
   "FIELD3": "203",
   "FIELD4": 303,
   "FIELD5": 403,
   "FIELD6": 503
 }
]